packages/server-wallet/src/wallet/store.ts

```diff
@@ -199,7 +201,30 @@ export class Store {
     channel = await timer('adding MY state', async () => this.addMyState(channel, signedState, tx));
 
     const sender = channel.participants[channel.myIndex].participantId;
-    const data = await timer('adding hash', async () => ({signedStates: [signedState]}));
+    let data: Payload = await timer('adding hash', async () => ({
+      signedStates: [signedState],
+      objectives: [],
+    }));
```

Comment:
General question (applies before this PR): 
- why is this being timed? 
- why is it labelled 'adding hash'? 
- why is it `async`?